api.py

```
@@ -0,0 +1,733 @@
+import argparse
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)